_docs/tasks/istio-auth.md

```diff
@@ -1,36 +1,162 @@
 ---
-title: Enabling Istio Auth
+title: Istio Auth
```

Comment:
I think you need a word like "Enabling" or "Installing" in the title. Otherwise, it's not particularly useful in the TOC.